releasenotes/notes/2.1/prepare-2.1.0-409d24ecbe277062.yaml

```diff
@@ -0,0 +1,28 @@
+---
+prelude: >
+  The Qiskit v2.1 release introduces several important enhancements in key areas including the C API, transpiler
+  capabilities and quantum circuit usability. In addition, it includes many general improvements and bug fixes.
+  The major highlights include:
+
+    * *C API extensions:* Building on the work started in the previous release, Qiskit v2.1 adds support for creating
+      and interacting with quantum circuits via C API functions, supporting the addition of standard gates, standard
+      instructions and unitary gates. Also added in this release is a set of C functions to build and manipulate a
+      :class:`.Target` object, as preparation for supporting a complete C API-based transpilation workflow in the next
+      release. In addition, :cpp:class:`QkComplex64` is now defined as a struct, serving as a compiler-agnostic representation
+      for complex numbers when working with Qiskit's C API. Note that the C API functions are generally compatible to
+      be used natively in C++.
```

Comment:
```suggestion
      for complex numbers when working with Qiskit's C API. The C API's header file has also been updated so that
      it is generally compatible to be used natively in C++.
```